app/views/admins/_user_entry.mobile.haml

```diff
@@ -0,0 +1,70 @@
+
+%li.user.media
+  %div.pull-left
+    - if user.person
+      %span.media-object
+        = person_image_tag(user.person, size: :thumb_small)
+
+  %div.media-body.row
```

Comment:
`%div.media-body.row` can be written as `.media-body.row` since `%div` is implicit